index.js

```diff
@@ -201,3 +201,48 @@ modal.addEventListener('click', (e) => {
 });
 
 generateProduct();
+
+// form validation
+const contactForm = document.querySelector('#contact_section');
+const fullName = document.querySelector('#name');
+const email = document.querySelector('#email');
+const textarea = document.querySelector('#textarea');
+const button = document.querySelector('#form_button');
+const messageErr = document.querySelector('#message');
+const checkLower = email.value;
+
+button.addEventListener('click', (event) => {
```

Comment:
According to the project requirements, the content of the email field has to be in lowercase. This feature has not been implemented correctly as seen in the video below:

https://user-images.githubusercontent.com/37116322/233440024-b9b36d39-1457-461f-a13c-3d4381e66882.mp4

Please take a look at your function and ensure the validation is done correctly